week-2/G-conditionals-2/exercise-2.js

```diff
@@ -8,6 +8,14 @@
 */
 
 function studentPassed(grade) {
+  if (grade<=49){
```

Comment:
It's probably easier to write `grade < 50` here since that's what the requirements state. The effect is the same though (assuming only whole numbers are used for marks). Otherwise what would happen if `grade` was `49.5`?